pootle/static/js/shared/components/KeyboardHelpDialog.js

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (C) Zing contributors.
+ *
+ * This file is a part of the Zing project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import React from 'react';
+import { showModal } from './Modal';
+
+import './KeyboardHelpDialog.css';
+
+let keyboardHelpOpen = false;
+
+export function showKeyboardHelpDialog() {
+  if (keyboardHelpOpen) {
+    return;
+  }
+
+  keyboardHelpOpen = true;
+  showModal({
+    title:
+      <span>
+        Keyboard Help (press <kbd>F1</kbd> to open;
```

Comment:
How about adding these under the _General_ section instead? I don't think these belong there, but I am happy to change my opinion if you show me examples of popular web apps that display such shortcuts in the title.